src/gpu/nvidia/cudnn_convolution_impl.hpp

```diff
@@ -65,7 +65,10 @@ struct cudnn_convolution_impl_base_t
     bool with_bias = false;
 
     bool do_scaling = false;
+    bool do_dst_scaling = false;
+    bool do_dst_scaling = false;
```

Comment:
This line breaks the build due to `do_dst_scaling` re-declaration, please fix

Reply:
It is fixed now. Thank you.